tests/verifyMongodbClusterSummaryDashboard_test.js

```diff
@@ -0,0 +1,16 @@
+
+Feature('to verify MongoDB Cluster Summary Dashboard');
+
+Before((I, loginPage) => {
+    I.amOnPage('/');
+    I.amOnPage(loginPage.url);
+    loginPage.login("admin", "admin");
+});
+
+Scenario('Open the MongoDB Cluster Summary Dashboard', async (I, adminPage, mondodbClusterSummaryPage) => {
+    I.amOnPage(mondodbClusterSummaryPage.url);
+    I.waitForElement(adminPage.fields.metricTitle, 30);
```

Comment:
Please check this, I am not waiting for MetricTitle, rather waiting for Header Panel found on each dashboard.